src/platform/datapath_raw_socket.c

```diff
@@ -265,10 +265,21 @@ CxPlatRemoveSocket(
     CxPlatRwLockReleaseExclusive(&Pool->Lock);
 }
 
+VOID
+CxPlatResolveRouteComplete(
+    _Inout_ CXPLAT_ROUTE* Route,
+    _In_ const uint8_t* PhysicalAddress
+    )
+{
+    CxPlatCopyMemory(&Route->NextHopLinkLayerAddress, PhysicalAddress, sizeof(Route->NextHopLinkLayerAddress));
+    Route->RouteState = RouteResolved;
+}
```

Comment:
I understand the desire to separate abstraction here, but IMO this is too much. No need to have a separate function for this when the caller has all the exposure already to do it themselves.

Reply:
This function will be called in 3 different places (2 now). Worth the abstraction?

Reply:
Still debatable for a 2 line function, but I'll leave it up to you.